paasta_tools/cli/cmds/spark_run.py

```diff
@@ -227,7 +255,7 @@ def configure_and_run_docker_container(
         return 1
     # Changes at docker ENTRYPOINT or CMD does not work.
     elif docker_cmd == 'jupyter':
-        docker_cmd = 'jupyter notebook -y --ip=%s --notebook-dir=%s --allow-root' % (
+        docker_cmd = 'jupyter notebook -y --ip=%s --notebook-dir=%s' % (
```

Comment:
Just out of curiosity, why did we need it before?

Reply:
Previously we didn't use --user for docker run. Root was the default user in the container to run jupyter.